docs/docs/adrs/adr-015-partial-set-security.md

```diff
@@ -192,33 +218,38 @@ Note that we only distribute rewards to validators that have been opted in for s
 ### Misbehaviour
 
 #### Fraud votes
-In an Opt In chain, a set of validators might attempt to perform an invalid-execution attack. To deter such potential attacks, PSS allows for the use of fraud votes.
-A _fraud vote_ is a governance proposal that enables the slashing of validators that performed an invalid-execution attack.
-Due to their inherent complexity, we intend to introduce fraud votes in a different ADR and at a future iteration of PSS. 
 
+In an Opt In chain, a set of validators might attempt to perform an attack. To deter such potential attacks, PSS allows for the use of fraud votes.
```

Comment:
Similarly to my previous [comment](https://github.com/cosmos/interchain-security/pull/1619/files#r1476256217), fraud votes deter a specific type of attack, so might be too vague if we remove the "invalid-execution" part.